gamemodes/terrortown/gamemode/client/cl_hud_manager.lua

```diff
@@ -193,6 +163,17 @@ function HUDManager.SetHUD(name)
 	net.SendToServer()
 end
 
+---
+-- Resets the current HUD if possible
+-- @realm client
+function HUDManager.ResetHUD()
+	local hud = huds.GetStored(HUDManager.GetHUD())
+
+	if not hud then return end
+
+	hud:Reset()
```

Comment:
This wont save any values by itself, so the reset might not be saved. 
Note this, as you also use it on the resolution change hook.